src/Faker/Core/Number.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Core;
+
+use Faker\Extension;
+
+/**
+ * @experimental This class is experimental and does not fall under our BC promise
+ */
+final class Number implements Extension\NumberExtension
+{
+    /**
+     * Returns a random number between $int1 and $int2 (any order)
+     *
+     * @param int $int1 default to 0
+     * @param int $int2 defaults to 32 bit max integer, ie 2147483647
+     *
+     * @example 79907610
+     */
+    public function numberBetween($int1 = 0, $int2 = 2147483647): int
```

Comment:
```suggestion
    public function numberBetween(int $int1 = 0, int $int2 = 2147483647): int
```

Reply:
Not

Reply:
@pimjansen I've renamed the arguments now, thanks!